scripts/generator/graphql_generator/csharp/Cart.CheckoutAccessors.cs.erb

```diff
@@ -5,6 +5,19 @@ namespace <%= namespace %> {
     using <%= namespace %>.GraphQL;
 
     public partial class Cart {
+
+        /// <summary>
+        /// Retrieve the currency code for the current checkout.
+        /// </summary>
+        /// <exception cref="NoCheckoutException">Calling this accessor before a checkout was created for the Cart. See Cart.GetWebCheckoutLink</exception>
+        public CurrencyCode GetCurrencyCode() {
```

Comment:
Exposing `CurrentCheckout` might mean we also need to expose `CheckoutSave`. Perhaps this is worth revisiting later? 

Making it return `null` might mislead the implementor to use these calls haphazardly, it should the programmer's responsibility to ensure these calls are made after a checkout has been created.